File: app/src/main/java/de/garten/training/depotflow/data/api/dto/OutboxDto.java
package de.garten.training.depotflow.data.api.dto;

public class OutboxDto {
    public String aggregateType;
    public String aggregateId;
    public String operation;
    public String payload;
    public String createdAt;

    public OutboxDto() {
    }

    public OutboxDto(String aggregateType, String aggregateId, String operation, String payload, String createdAt) {
        this.aggregateType = aggregateType;
        this.aggregateId = aggregateId;
        this.operation = operation;
        this.payload = payload;
        this.createdAt = createdAt;
    }
}

File: app/src/main/java/de/garten/training/depotflow/data/api/dto/OutboxDto.kt
package de.garten.training.depotflow.data.api.dto

class OutboxDto {
    var aggregateType: String? = null
    var aggregateId: String? = null
    var operation: String? = null
    var payload: String? = null
    var createdAt: String? = null

    constructor()

    constructor(
        aggregateType: String?,
        aggregateId: String?,
        operation: String?,
        payload: String?,
        createdAt: String?
    ) {
        this.aggregateType = aggregateType
        this.aggregateId = aggregateId
        this.operation = operation
        this.payload = payload
        this.createdAt = createdAt
    }
}
